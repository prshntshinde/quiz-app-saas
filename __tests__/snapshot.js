import { render } from "@testing-library/react";
import Home from "../src/app/page.tsx";

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
