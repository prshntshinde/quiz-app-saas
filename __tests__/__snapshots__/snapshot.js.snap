// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex max-h-screen w-full flex-col"
  >
    <header
      class="sticky top-0 flex h-16 items-center gap-4 border-b bg-background px-4 md:px-6"
    >
      <nav
        class="hidden flex-col gap-6 text-lg font-medium md:flex md:flex-row md:items-center md:gap-5 md:text-sm lg:gap-6"
      >
        <a
          class="flex items-center gap-2 text-lg font-bold md:text-base"
          href="/"
        >
          <svg
            class="lucide lucide-school h-6 w-6"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M14 22v-4a2 2 0 1 0-4 0v4"
            />
            <path
              d="m18 10 4 2v8a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2v-8l4-2"
            />
            <path
              d="M18 5v17"
            />
            <path
              d="m4 6 8-4 8 4"
            />
            <path
              d="M6 5v17"
            />
            <circle
              cx="12"
              cy="9"
              r="2"
            />
          </svg>
        </a>
        <a
          class="hover:bg-sky-100 hover:text-blue-600 transition ease-in-out delay-50 hover:-translate-y-1 hover:scale-110 hover:duration-300 font-bold"
          href="/"
        >
          Home
        </a>
        <a
          class="hover:bg-sky-100 hover:text-blue-600 transition ease-in-out delay-50 hover:-translate-y-1 hover:scale-110 hover:duration-300 font-bold"
          href="/quiz"
        >
          Quiz
        </a>
      </nav>
      <button
        aria-controls="radix-:r0:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 border border-input bg-background hover:bg-accent hover:text-accent-foreground h-10 w-10 shrink-0 md:hidden"
        data-state="closed"
        type="button"
      >
        <svg
          class="lucide lucide-menu h-5 w-5"
          fill="none"
          height="24"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <line
            x1="4"
            x2="20"
            y1="12"
            y2="12"
          />
          <line
            x1="4"
            x2="20"
            y1="6"
            y2="6"
          />
          <line
            x1="4"
            x2="20"
            y1="18"
            y2="18"
          />
        </svg>
        <span
          class="sr-only"
        >
          Toggle navigation menu
        </span>
      </button>
      <div
        class="invisible flex flex-row-reverse w-full items-center md:visible"
      >
        <a
          class="hover:bg-sky-100 hover:text-blue-600 transition ease-in-out delay-50 hover:-translate-y-1 hover:scale-110 hover:duration-300 font-bold"
          href="/admin"
        >
          Admin
        </a>
      </div>
    </header>
    <main
      class="flex"
    >
      <div
        class="flex-col max-w-full mx-auto"
      >
        <img
          alt="Quiz App"
          data-nimg="1"
          decoding="async"
          height="400"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=1200&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=640&q=75 1x, /_next/image?url=%2Fimg.jpg&w=1200&q=75 2x"
          style="color: transparent;"
          width="600"
        />
      </div>
    </main>
  </div>
</div>
`;
